scipy/spatial/distance.py

```diff
@@ -544,9 +544,7 @@ def sqeuclidean(u, v, w=None):
 
     .. math::
 
-       {\\|u-v\\|}_2^2
-
-       \\left(\\sum{(w_i |(u_i - v_i)|^2)}\\right)
+       \\sum\\limits_i w_i {||u-v||}_2^2
```

Comment:
What happened to the i subscripts for u and v?

Reply:
Agree I prefer Warren's suggestion of `\sum_i{w_i |u_i - v_i|^2}`

Reply:
I have updated the equation to use Warren's suggestion and added back in the subscripts for u and v. Thanks for the feedback!